tasks/level-1/1.8.yml

```diff
@@ -7,6 +7,7 @@
   yum:
     name: "*"
     state: latest
+    exclude: nginx*
```

Comment:
does it need *?

Reply:
Yeah, I want to wildcard here in case there are some other nginx deps that shouldn't be updated in tandem. The examples are also all wildcarded, I'm guessing for the same reason.